### PR TITLE
Try to run tests on crowdin PR creation

### PR DESCRIPTION
**Pull request by wojsmol**
Try to run tests on crowdin PR'a and eliminate need  of closing and reopening of normal PR to get tests results.

### CODE REVIEW

**Review by tecpromotion — APPROVED** [review]

**Comment by tecpromotion** [comment]
Ready to merge @wojsmol ?

**Comment by wojsmol** [comment]
@tecpromotion In https://github.com/joomla/core-translations/pull/392/commits/53babef51c2252e3ba2385803578c8b3133c35c5 I fixed type name - deference between external article end official GH docs https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

File: .github/workflows/_check-php-code.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ on:
   pull_request:
     branches:
       - main
+    types: [synchronize, opened ]
```
Comment (tecpromotion) [inline .github/workflows/_check-php-code.yml:7]:
```suggestion
    types: [synchronize, opened]
```

File: .github/workflows/_en-gb-localise.yml [hunk 2]
```diff
@@ -4,6 +4,7 @@ on:
   pull_request:
     branches:
       - main
+    types: [synchronize, opened ]
```
Comment (tecpromotion) [inline .github/workflows/_en-gb-localise.yml:7]:
```suggestion
    types: [synchronize, opened]
```

**Review by tecpromotion — CHANGES_REQUESTED** [review]
just a space

**Review by tecpromotion — APPROVED** [review]

**Pull request merged**